### ISSUE TITLE
Accessibilty statement

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#13]
There should be an accessibility statement linked from the pol.is page

* Based on the Cabinet Office a11y statement
* Informed by latest a11y report

**Comment by patcon** [comment]
Fuckin _love_ this. (Can I say that on a government repo?? 😁 )

On the https://talent.canada.ca team, we use https://makeitfable.com/ for equitable accessibility testing with an org that cares deeply about employing people living with disabilities. Might there be political will to fund a subscription to them? They give really structured feedback that would be rad to upstream!

Related: https://github.com/DFE-Digital/polis-whitelabel/issues/8

**Comment by patcon** [comment]
Happy to set up and/or hop on a call with Fable reps, if you're interested to hear more :)

cc: @katekalcevich @LaurentGoderre

**Comment by patcon** [comment]
maybe worth adding `a11y` label?

**Comment by brendanarnold** [comment]
Hi thanks for the recommendation @patcon - we had an internal report completed in October this year by our accessibility specialists at the Department for Education which I've reflected in the placeholder statement. I hope to be in discussion later with them to follow up on this when we have some of the remedies in place.
